### ISSUE TITLE
Use React/Redux/Re-frame

### ISSUE THREAD

**Issue by binarykitchen** [binarykitchen/videomail-client#19]
I see a huge potential here to use React!!!!

**Comment by omnipotentuser** [comment]
Elaborate? :D

**Comment by binarykitchen** [comment]
The videomail-client code is a crazy state machine.

If you look in the code, you see I have lots of flags controlling behavior of many functions and it's getting bloated soon. The current two-way-data binding and event handling mechanism isn't optimal. That's exactly one thing React is very good at. Just seeing a huge potential here to write much more elegant code with React, especially for maintainability's sake.

**Comment by binarykitchen** [comment]
Maybe React Native!

**Comment by binarykitchen** [comment]
Nope, it should be either Redux or in ClojureScript using Reagent/re-frame.

**Comment by binarykitchen** [comment]
Outdated ticket. Closed.
